client/gutenberg/extensions/slideshow/slideshow.js

```diff
@@ -8,8 +8,12 @@ import { isEqual } from 'lodash';
  * Internal dependencies
  */
 import createSwiper from './create-swiper';
+import ResizeObserver from 'resize-observer-polyfill';
```

Comment:
Nitpick -- That's an _external_ dependency :grimacing:

Reply:
I was confused by https://github.com/Automattic/wp-calypso/blob/master/client/gutenberg/extensions/tiled-gallery/view.js#L5 which has it in Internal, but looking closer I see it elsewhere under External. I'll fix this.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/30736/commits/145251548eb6822af54551c84354d10c17de63d5